benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm

```diff
@@ -0,0 +1,181 @@
+# This parameter file reproduces a 2D benchmark for viscoelastic deformation
+# published in "Numerical modelling of magma dynamics coupled to tectonic
+# deformation of lithosphere and crust" by Keller, May and Kaus, 2013, 
+# Geophys. J. Int., v. 195,  p. 1406-1422. The model setup and results are
+# described in detail in section B.2.3 and figure B5.
+#
+# The benchmark examines bending and unbending (e.g., recovery) of a 
+# viscoelastic beam surrounded by a less dense and viscous (inelastic) fluid. 
+# Gravitational forces drive initial bending (elastic strain) of the beam for
+# 50 Kyr, which then recovers its shape over ~ 500 Kyr if gravity is turned 
+# off. The recovery is driven by the accumulated elastic stresses and thus
+# provides a basic test for the viscoelasticity implementation. 
+#
+# Compositional fields are used to track the viscoelastic stresses and material 
+# representing the beam. To improve the accuracy of tracking the beam interface,
+# a discontinuous discretization and bound preserving limiter is used for the 
+# compositional fields. Significantly, the time step is limited to 1 Kyr through
+# the "Maximum time step" parameter and a relatively high (0.5) CFL value. Using
+# a constant time step ensures the effective (viscoelastic) viscosity of the
+# viscoelastic beam and viscous fluid remains constants throughout the model run.
```

Comment:
Does that mean the solution is time step dependent otherwise? Is there a separate elastic timestep from the convection timestep?